cylc/flow/network/scan.py

```diff
@@ -21,6 +21,7 @@
 import re
 import sys
 import socket
+import logging
```

Comment:
I think instead we can use `from cylc.flow import LOG`. Then we don't need the line configuring the logging.

Reply:
Done! :)